go.mod

```diff
@@ -60,16 +58,15 @@ require (
 	github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e // indirect
 	github.com/cockroachdb/apd/v2 v2.0.2 // indirect
 	github.com/coinbase/rosetta-sdk-go v0.7.9 // indirect
-	github.com/confio/ics23/go v0.7.0 // indirect
+	github.com/confio/ics23/go v0.9.0 // indirect
```

Comment:
Like recommended by cosmos-sdk [release note for v0.46.7](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.46.7), since `github.com/confio/ics23/go` is on v0.9.0, we could remove the replace directive from `go.mod` file : 
`github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0`